### PR TITLE
Adding RN for Platform 5.3.5

### PR DESCRIPTION
**Pull request by Serdaro**

### CODE REVIEW

File: docs/modules/release-notes/pages/5-3-3.adoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+* Changed the exception type from `CancellationException` to `CancellationByUserException` in case the user cancels a job before it is initialized.
+https://github.com/hazelcast/hazelcast/pull/25452[#25452]
+* Updated the gRPC dependency version to 1.57.0.
+https://github.com/hazelcast/hazelcast/pull/25430[#25430]
+* Renamed the service port for Hazelcast clusters deployed in Kubernetes environments as `hazelcast`.
+Previously, the name was `hazelcast-service-port` causing the member auto-discovery (for embedded deployments) to fail.
+https://github.com/hazelcast/hazelcast/pull/24834[#24834]
+
+== Fixes
```
Comment (frant-hartm) [inline docs/modules/release-notes/pages/5-3-3.adoc:13]: This PR is notable from my POV https://github.com/hazelcast/hazelcast/pull/25661

Comment (kwart) [reply docs/modules/release-notes/pages/5-3-3.adoc:13]: and other permission related improvements in https://github.com/hazelcast/hazelcast/pull/25539

File: docs/modules/release-notes/pages/5-3-3.adoc [hunk 2]
```diff
@@ -0,0 +1,20 @@
+= 5.3.3 Release Notes
+
+== Enhancements
+
+* Changed the exception type from `CancellationException` to `CancellationByUserException` in case the user cancels a job before it is initialized.
+https://github.com/hazelcast/hazelcast/pull/25452[#25452]
+* Updated the gRPC dependency version to 1.57.0.
```
[the remark this answers is not in the document]
Comment (Serdaro) [reply docs/modules/release-notes/pages/5-3-3.adoc:7]: Updating these @frant-hartm , but I couldn't see the elasticsearch for 5.3.3. There is a merged PR for 5.4.0: https://github.com/hazelcast/hazelcast/pull/25394

File: docs/modules/release-notes/pages/5-3-3.adoc [hunk 3]
```diff
@@ -0,0 +1,39 @@
+= 5.3.3 Release Notes
+
+== Enhancements
+
+* Improved the permission checks in the file connectors by adding a method that returns the permissions required to resolve field names.
```
Comment (k-jamroz) [inline docs/modules/release-notes/pages/5-3-3.adoc:5]: this is very diplomatic way of describing that a vulnerability was fixed, should we link CVE here?

Comment (Serdaro) [reply docs/modules/release-notes/pages/5-3-3.adoc:5]: @k-jamroz , I couldn't find the CVE link in the relevant PRs or the Jira ticket. Is there a CVE ID?

Comment (k-jamroz) [reply docs/modules/release-notes/pages/5-3-3.adoc:5]: probably not yet: HZ-3405

**Review by vbekiaris — APPROVED** [review]

**Review by k-jamroz — APPROVED** [review]

**Review by frant-hartm — APPROVED** [review]

**Review by kwart — APPROVED** [review]

**Pull request merged**